pymedphys/_mosaiq/constants.py

```diff
@@ -35,3 +35,16 @@
     20: "User Defined",
     21: "PDR",
 }
+
+ORIENTATION = {
+    0: "Unknown",
+    1: "HFS",
+    2: "HFP",
+    3: "HFRL",
```

Comment:
If this is from DICOM, the defined values in DICOM are HFDL (Ducubitus Left) and HFDR (Decubitus Right), and similar for FF.  I'm not saying this is necessarily incorrect, but the others line up with DICOM.
See http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.3.html#sect_C.7.3.1.1.2 

  (other than "Unknown", and the Patient Position element (0x0018, 0x5100) has Sitting as SITTING (capitalized), see
http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.8.12.html#sect_C.8.8.12.1.2

Reply:
@rembishj before merging can you make these align with the DICOM standard?